contracts/ERC721ApprovalEnumerable.sol

```diff
@@ -0,0 +1,91 @@
+pragma solidity 0.5.8;
```

Comment:
Naming was a bit of a struggle here so if anyone has better names for things, please comment.